Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -124,4 +133,33 @@ namespace PhysX
             EXPECT_THAT(expectedGravityMultiplier + i, testing::FloatEq(basis.m_gameplayController->GetGravityMultiplier()));
         }
     }
+
+    TEST_F(PhysXDefaultWorldTest, CharacterGameplayController_EntityFallsUnderGravity)
+    {
+        GameplayTestBasis basis(m_testSceneHandle, DefaultGravityMultiplier, DefaultGroundDetectionBoxHeight, DefaultFloorTransform);
+        auto* sceneInterface = AZ::Interface<AzPhysics::SceneInterface>::Get();
+        sceneInterface->RemoveSimulatedBody(basis.m_sceneHandle, basis.m_floor->m_bodyHandle);
+        
+        // Let scene run for a few moments so the entity can be manipulated by gravity from the gameplay component
+        auto startPosition = basis.m_controller->GetPosition();
+        
+        int timeStepCount = 10;
+        float totalTime = 0.0f;
+        float timeStep = AzPhysics::SystemConfiguration::DefaultFixedTimestep;
+        float distanceFell = 0.0f;
```

Comment:
- `startPosition` can be const.
- `timeStepCount` can be const
- `timeStep` can be const.
- `distanceFell` is not used until later so it can be moved where it's needed and be const.
````
const float distanceFell = 0.5f * (-10.0f) * (totalTime * totalTime);
`````

Reply:
Udpated